mllib/src/main/scala/org/apache/spark/ml/evaluation/BinaryClassificationEvaluator.scala

```diff
@@ -29,6 +29,7 @@ import org.apache.spark.sql.types.DoubleType
 /**
  * :: Experimental ::
  * Evaluator for binary classification, which expects two input columns: rawPrediction and label.
+ * The rawPrediction column can be of type double or vector.
```

Comment:
How about a little more explicit:
```The rawPrediction column can be of type double (binary 0/1 prediction, or probability of label 1) or of type vector (length-2 vector of raw predictions, scores, or label probabilities).```

Also, could you please update the Python class doc with the same text?